src/Methods/Methods.tsx

```diff
@@ -31,8 +31,8 @@ const styles = (theme: Theme) => ({
     fontSize: theme.typography.pxToRem(15),
   },
   root: {
-    marginBottom: theme.spacing.unit,
-    marginTop: theme.spacing.unit * 3,
+    marginBottom: theme.spacing(2),
+    marginTop: theme.spacing(3),
```

Comment:
should this require a multiplier as well?

Reply:
discussed offline. Theme update shifts spacing to a function, so the multiplication no longer seems necessary for the newer rendering methods.